src/App.js

```diff
@@ -0,0 +1,24 @@
+import { Route, Routes } from 'react-router-dom';
+import { useDispatch } from 'react-redux';
+
+import Home from './components/Home';
+import Details from './components/Details';
+
+import { getSymbols } from './slices/CompanySymbolSlice';
+import Header from './components/Header';
+
+function App() {
+  const dispatch = useDispatch();
+  dispatch(getSymbols());
+  return (
+    <div className="App">
+      <Header />
+      <Routes>
+        <Route path="/" element={<Home />} />
+        <Route path="/details/:symbol/:index" element={<Details />} />
+      </Routes>
+    </div>
+  );
+}
```

Comment:
- [ ] Good job so far @Abdessittir, but I think your app has some issues. When I run the app locally I get this 👇 👇 

<img width="526" alt="Screen Shot 2023-01-04 at 22 43 35" src="https://user-images.githubusercontent.com/74501523/210638827-e6e4eccb-cbbc-4b30-be1f-d26a0da53e56.png">

- [ ] Your live demo link works correctly but when we run the app locally it not working, please kindly fix that. I you have some information about how to run your app locally or about the API documentation kindly add it to your README file then everyone can be able to run your app locally 🙃 😉